### PR TITLE
Let integration tests generic over externs

### PR DESCRIPTION
**Pull request by mriise**
@tchataigner

### CODE REVIEW

**Comment by tchataigner** [comment]
Thank you for that it's awesome ! Sorry for the delay :( Opened a [PR](https://github.com/mriise/ref-fvm/pull/2) on yours to add mock behavior to those dummies 👍

**Comment by mriise** [comment]
thanks! @tchataigner sorry for the delay as well :\

File: testing/integration/examples/integration.rs [hunk 1]
```diff
@@ -50,7 +51,7 @@ pub fn main() {
         .unwrap();
 
     // Instantiate machine
-    tester.instantiate_machine().unwrap();
+    tester.instantiate_machine(DummyExterns).unwrap();
```
Comment (raulk) [inline testing/integration/examples/integration.rs:54]: Maybe we could do some magic here with a default implementation for the Externs type param, but this is good enough ;-)

**Review by raulk — APPROVED** [review]

**Review by tchataigner — APPROVED** [review]
I think it does the job ! Thank you for handling it !

**Pull request merged**